src/app/calculator/furnaces/energy-use/energy-use.component.ts

```diff
@@ -160,26 +161,6 @@ export class EnergyUseComponent implements OnInit {
     }
   }
 
-  saveCalculator() {
-    if (!this.saving || this.calcExists) {
-      if (this.calcExists) {
-        this.indexedDbService.putCalculator(this.calculator).then(() => {
-          this.calculatorDbService.setAll();
-        });
-      } else {
-        this.saving = true;
-        this.calculator.assessmentId = this.assessment.id;
-        this.indexedDbService.addCalculator(this.calculator).then((result) => {
-          this.calculatorDbService.setAll().then(() => {
-            this.calculator.id = result;
-            this.calcExists = true;
```

Comment:
I think the `calcExists` logic of being turned to `true` when a new calculator is added is being lost here